src/main/resources/default/templates/biz/tycho/analytics/data-explorer.html.pasta

```diff
@@ -0,0 +1,466 @@
+                                    class="range-select-js btn btn-outline-primary"
+                                    data-range="last90Days"
+                                    onclick="changeRange('last90Days')">
+                                @i18n("DataExplorerController.last90Days")
+                            </button>
+                            <button type="button"
+                                    class="range-select-js btn btn-outline-primary"
+                                    data-range="lastMonth"
+                                    onclick="changeRange('lastMonth')">
+                                @i18n("DataExplorerController.lastMonth")
+                            </button>
+                            <button type="button"
+                                    class="range-select-js btn btn-outline-primary"
+                                    data-range="lastYear"
+                                    onclick="changeRange('lastYear')">
+                                @i18n("DataExplorerController.lastYear")
+                            </button>
+                        </div>
+                    </div>
+                    <div class="col-12 col-md-6 mb-2">
+                        <div class="mb-2 text-small text-muted">@i18n("DataExplorerController.comparisonPeriod")</div>
+                        <div class="btn-group">
+                            <button type="button"
+                                    class="period-select-js btn btn-outline-primary"
+                                    data-period="none"
+                                    onclick="changePeriod('none')">@i18n("DataExplorerController.periodOff")
+                            </button>
+                            <button type="button"
+                                    class="period-select-js btn btn-primary"
+                                    data-period="lastYear"
+                                    onclick="changePeriod('lastYear')">@i18n("DataExplorerController.periodLastYear")
+                            </button>
+                            <button type="button"
+                                    class="period-select-js btn btn-outline-primary"
+                                    data-period="lastMonth"
+                                    onclick="changePeriod('lastMonth')">@i18n("DataExplorerController.periodLastMonth")
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div id="charts">
+
+        </div>
+    </div>
+
+    <script type="text/javascript">
+        let range = 'last12Months';
+        let comparisonPeriod = 'lastYear';
+
+        // Maps typs of charts to their actual JS callbacks...
+        const CHART_TYPES = {
+            'line': lineChartType,
+            'area': areaChartType
+        };
+
+        // Tries to find the chart DIV with the given identifier (TYPE or TYPE:OBJECT).
```

Comment:
On the server-side the identifier seems to always be split (TYPE:OBJECT). Is this comment correct?

Reply:
A split is attempted and TYPE will result int (TYPE, null) whereas TYPE:OBJECT will result int (TYPE, OBJECT)